packages/xo-server-backup-reports/src/index.js

```diff
@@ -192,9 +192,11 @@ class BackupReportsXoPlugin {
       ...(await templates.markdown.transform(templates.markdown.$metadata(context))),
       ...(await templates.compactMarkdown.transform(templates.compactMarkdown.$metadata(context))),
       ...(await templates.mjml.transform(templates.mjml.$metadata(context))),
+      ...(await templates.compactMjml.transform(templates.compactMjml.$metadata(context))),
       mailReceivers,
       subject: templates.mjml.$metadataSubject(context),
       success: log.status === 'success',
+      shorterBackupReports: log.data?.shorterBackupReports,
```

Comment:
`compactBackupReport`

Reply:
In the job's settings, use an explicit template name instead of a boolean for future proofing:

```js
{
  backupReportTpl: compact ? 'compactMjml' : 'mjml',
}
```